components/cart/add-to-cart.tsx

```diff
@@ -54,25 +51,21 @@ export function AddToCart({
             return;
           }
 
-          setCookie('cartId', response, {
```

Comment:
**NOTE:** I've removed setCookie here since it's been added in `actions/addItem`